internal/iot/client.go

```diff
@@ -85,7 +90,8 @@ func (cl *Client) DeviceLoraCreate(ctx context.Context, name, serial, devType, e
 		Type:          devType,
 		UserId:        "me",
 	}
-	dev, _, err := cl.api.LoraDevicesV1Api.LoraDevicesV1Create(ctx, payload)
+
+	dev, _, err := cl.api.LoraDevicesV1Api.LoraDevicesV1Create(ctx, payload, nil)
```

Comment:
What is that additional `nil` parameter?

Reply:
We recently introduced the possibility to set the x-organization header also for Lora devices (previously was not managed for Lora devices). This is reflected in the swagger as an optional header. The auto-generated go client uses to add an optional parameter in this case. As the XOrganization header is managed in a different way by the cloud-cli, if any other optional parameter is available for the endpoint, a nil value is passed. This is how the same situation is currently managed in other similar situations:
Eg.
[here](https://github.com/arduino/arduino-cloud-cli/blob/b79dd32a0c6b9310951d1edb213f07f84a42c2de/internal/iot/client.go#L63)
and 
[here ](https://github.com/arduino/arduino-cloud-cli/blob/b79dd32a0c6b9310951d1edb213f07f84a42c2de/internal/iot/client.go#L414)